README.md

```diff
@@ -23,21 +23,30 @@ Elastic search for Sylius.
     $ elasticsearch
     ```
 
-4. Add this bundle to `AppKernel.php`:
+4. Add those bundles to `AppKernel.php`:
 
     ```php
      new \ONGR\ElasticsearchBundle\ONGRElasticsearchBundle(),
      new \SimpleBus\SymfonyBridge\SimpleBusCommandBusBundle(),
      new \SimpleBus\SymfonyBridge\SimpleBusEventBusBundle(),
      new \ONGR\FilterManagerBundle\ONGRFilterManagerBundle(),
+     new \Sylius\ElasticSearchPlugin\SyliusElasticSearchPlugin(),
     ```
 
 5. Create/Setup database:
 
     ```bash
-    $ app/console do:da:cr
-    $ app/console do:sch:cr
-    $ app/console syl:fix:lo
+    $ bin/console ongr:es:index:create
+    $ bin/console do:da:cr
+    $ bin/console do:sch:cr
+    $ bin/console syl:fix:lo
+    ```
+
+    If there is a problem with creating elastic search index run those commands:
```

Comment:
`If there is a problem with creating elastic search index, try dropping the previous one and recreate it with those commands`?